docs/make.jl

```diff
@@ -10,6 +10,84 @@ if haskey(ENV, "DOCSARGS")
     end
 end
 
+# ==============================================================================
+#  Modify the release notes
+# ==============================================================================
+
+function fix_release_line(
+    line::String;
+    repo::String = "JuliaDocs/Documenter.jl",
+)
+    # Rule: ((abc#XXXX) -> ([abc#XXXX](https://github.com/abc/issue/XXXX))
+    # Description: Replace issue/PR nnumbers with a link to the default repo
+    # Example: (JuliaLang/julia#123) -> ([JuliaLang/julia#123](https://github.com/JuliaLang/julia/issues/123))
+    # There is no need to distinguish between PRs and Issues because GitHub
+    # redirects.
+    while (m = match(r"\(([a-zA-Z0-9/]+?)\#([0-9]+)\)", line)) !== nothing
+        new_repo, id = m.captures[1], m.captures[2]
+        line = replace(line, m.match => "([$new_repo#$id](https://github.com/$new_repo/issues/$id))")
+    end
+    # Rule: (#XXXX) -> ([#XXXX](https://github.com/url/issue/XXXX))
+    # Description: Replace issue/PR nnumbers with a link to the default repo
+    # Example: (#123) -> ([#123](https://github.com/JuliaDocs/Documenter.jl/issues/123))
+    # There is no need to distinguish between PRs and Issues because GitHub
+    # redirects.
+    while (m = match(r"\(\#([0-9]+)\)", line)) !== nothing
+        id = m.captures[1]
+        line = replace(line, m.match => "([#$id](https://github.com/$repo/issues/$id))")
```

Comment:
Documenter actually handles references issue references to the current repo, so this should be sufficient:

```suggestion
        line = replace(line, m.match => "([#$id](#$id))")
```